app/code/core/Mage/AdminNotification/Model/Feed.php

```diff
@@ -55,7 +57,7 @@ public function getFeedUrl()
      */
     public function checkUpdate()
     {
-        if (($this->getFrequency() + $this->getLastUpdate()) > time()) {
+        if (($this->getFrequency() + $this->getLastUpdate()) > Carbon::now()->getTimestamp()) {
```

Comment:
i meant this line

wouldn't it be better if it would call:
```
Mage::helper('core')->getClock()->now()->getTimestamp()
```
or hiding the clock internal:
```
Mage::helper('core')->now()->getTimestamp()
```

with that we can use PsrClock (and their implementations) for Test-Mocks too

Reply:
Yep!!!! :+1: 

Again, this PR is about to make carbon available for further actions.